
2022-10-23 18:07:57.450 - [Debug] [Connection] Attacker connected: 120.236.74.234 | Client Identification: SSH-2.0-Go
2022-10-23 18:07:57.715 - [Debug] [Connection] Attacker connected: 120.224.143.251 | Client Identification: SSH-2.0-Go
2022-10-23 18:07:57.771 - [Debug] [Connection] Attacker connected: 124.220.163.75 | Client Identification: SSH-2.0-Go
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
(node:47045) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'DATABASE_1',
  containerIp: '10.0.3.43',
  mitmPort: 1000,
  autoAccessFixed: 1
}
2022-10-23 18:08:32.859 - [Info] MITM Version: 2
2022-10-23 18:08:32.860 - [Info] Auto Access Enabled: true
2022-10-23 18:08:32.860 - [Debug] [Init] Auto Access Theshold Achieved: false
2022-10-23 18:08:32.867 - [Info] SSH man-in-the-middle server for 10.0.3.43 listening on 10.0.3.1:1000
2022-10-23 18:08:33.354 - [Debug] [Connection] Attacker connected: 59.1.226.211 | Client Identification: SSH-2.0-Go
2022-10-23 18:08:33.612 - [Debug] [Connection] Attacker connected: 84.16.224.173 | Client Identification: SSH-2.0-Go
2022-10-23 18:08:34.202 - [Debug] [Auth] Attacker 84.16.224.173 trying to authenticate with 'none'
2022-10-23 18:08:34.203 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-23 18:08:34.426 - [Debug] [Auth] Attacker 84.16.224.173 trying to authenticate with 'publickey'
2022-10-23 18:08:34.427 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-23 18:08:34.428 - [Debug] [Auth] Attacker: 84.16.224.173 has so far made 1 attempts. Remaining: 5 attempts
2022-10-23 18:08:34.651 - [Debug] [Connection] Attacker closed the connection
2022-10-23 18:08:34.685 - [Debug] [Connection] Attacker connected: 101.43.224.103 | Client Identification: SSH-2.0-Go
2022-10-23 18:08:34.747 - [Debug] [Connection] Attacker connected: 84.16.224.173 | Client Identification: SSH-2.0-Go
